src/components/card/Card.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <div class="card">
+        <header class="card-header" :class="{'is-collapsible' : collapsible}" @click="toggle">
+            <p class="card-header-title" v-if="header" v-html="header"></p>
+            <slot v-else name="header"></slot>
+            <a class="card-header-icon">
+                <span class="icon">
+                    <b-icon v-if="collapsible" 
```

Comment:
Make sure to import `b-icon` and register it below in case developers aren't importing the whole Buefy library.